docs/CallingConvention.rst

```diff
@@ -12,10 +12,10 @@ want it to be.
 
 It's a basic assumption in this paper that Swift shouldn't make an
 implicit promise to exactly match the default platform calling
-convention.  That is, if a C or ObjC programmer manages to derive the
+convention. That is, if a C or Objective C programmer manages to derive the
```

Comment:
If we are going to remove the abbreviation, we should use the standard spelling which is "Objective-C".